.github/styles/Vocab/Train/accept.txt

```diff
@@ -1,5 +1,5 @@
 Horovod
-Hugging Face
+hyperparameters?
```

Comment:
do we need to have the question mark here?

Reply:
I believe this is a regex that matches "hyperparameter" and "hyperparameters"

Reply:
The quotation mark means the "s" is optional. 

You might also want to add a "$" to the end (`hyperparameters?$`) to avoid matching words like "hyperparametere"